src/stateProviders/provideFenceConnections.js

```diff
@@ -46,7 +49,21 @@ export default provideState({
       const fenceConnectionsFetchArray = FENCES.map(fence =>
         getFenceUser(api, fence)
           .then(details => {
-            effects.addFenceConnection({ fence, details });
+            effects.addFenceConnection({
+              fence,
+              details,
+            });
+            // Now we also need to get studies for this fence
+            return getUserStudyPermission(api, {
+              [fence]: details,
+            })({})
+              .then(({ acceptedStudiesAggs, unacceptedStudiesAggs }) =>
+                effects.addFenceStudies(fence, {
+                  authorizedStudies: acceptedStudiesAggs,
+                  unauthorizedStudies: unacceptedStudiesAggs,
+                }),
+              )
+              .catch(err => console.log(`Error fetching fence studies for '${fence}': ${err}`));
```

Comment:
if one fails, the `Promise.all` will reject. I the dashboard card isn't able to handle that eh

Reply:
👍